.github/workflows/templates-api.yml

```diff
@@ -24,28 +25,70 @@ jobs:
         uses: actions/checkout@v3
         with:
           ref: ${{ github.event.pull_request.head.ref }}
+
+      - name: Create artifacts directory
+        run: mkdir -p /tmp/artifacts
+
+      - name: Install gron tool
+        run: |
+          url="https://github.com/tomnomnom/gron/releases/download/v0.7.1/gron-linux-amd64-0.7.1.tgz"
+          curl -L "$url" | tar -xz -C /usr/local/bin
+
       - name: Start MiniKube
         run: ./start_minikube.sh
+
       - name: Set Kubernetes namespace
         run: kubectl config set-context --current "--namespace=$KUBERNETES_NAMESPACE"
+
       - name: Checkout BASE branch
         uses: actions/checkout@v3
         with:
           ref: ${{ github.event.pull_request.base.ref }}
-      - name: Deploy the previous version, from the BASE branch
+
+      - name: Deploy the old version, from the BASE branch
+        continue-on-error: true
         run: ./deploy_local.sh
+
+      - name: Dump the old version (for diff)
+        continue-on-error: true
+        run: |
+          set -Eeuo pipefail
+          # Wait for pod startup/termination
+          sleep 10
+          # List objects
+          kubectl get all --namespace templates-api
+          # Dump objects (kubectl get all doesn't return all types)
+          kubectl get all,configmaps,pdb --ignore-not-found --sort-by='{.metadata.name}' --output json \
```

Comment:
`kubectl get all` dumpne objekty ako `json`, ... `all` nie je `all` a niektore typy to neobsahuje :slightly_smiling_face: , tak som doplnil co pouzivame.